packages/teams-js/test/public/media.spec.ts

```diff
@@ -655,11 +505,9 @@ describe('media', () => {
             expect(mediaAttachments).toBeFalsy();
```

Comment:
Do we need to use expect.assertions in this function to make sure that the expect statements inside of the callback are hit? (I'm not sure off the top of my head, legit question)

Reply:
since i didn't touch this function, i can change this in the next ticket regarding fixing the tests in this file